src/main/kotlin/util/StringExt.kt

```diff
@@ -53,3 +57,7 @@ fun String.Companion.unHex(s: String): ByteArray {
 fun String.Companion.irohaEscape(text: String): String {
     return text.replace("\"", IROHA_FRIENDLY_QUOTE)
 }
+
+//TODO can we get rid of klaxon and moshi? Gson is much easier thing to use.
```

Comment:
I propose to create a task in backlog for that

Reply:
IMO Unification of the tools used in a project is always a good proposal. We can define a potential task without specifying the concrete library, but list of possible candidates and vote afterwards after discussion who is better and why.

Reply:
I think we can use https://github.com/Kotlin/kotlinx.serialization. It is shipped with Kotlin since 1.3. Also I've added a task to migrate.